cli/src/command/wallet.rs

```diff
@@ -168,7 +168,7 @@ pub async fn init_command(
         .await?)
 }
 
-pub async fn list_accounts_command(storage_path: &Path, snapshot_path: &Path) -> Result<(), Error> {
+pub async fn accounts_command(storage_path: &Path, snapshot_path: &Path) -> Result<(), Error> {
```

Comment:
I feel like `list_accounts` is better...

Reply:
Honestly, me too. It's much clearer to have Verb+Noun in the variant and in the corresponding fn. However, for some reasons we decided to call the variant `Accounts`, so the name of this fn is kinda pre-determined if we want be consistent with the rest of the file.

Reply:
Can we change both? I may have missed a discussion but I feel like both should be `list_accounts`.

Reply:
@thibault-martinez ^

Reply:
I wouldn't, no. None of the other commands that list have a List prefix, not sure why we'd do it for this one. Happy to discuss it for 2.0 though.